packages/next/src/server/next-server.ts

```diff
@@ -1322,6 +1325,12 @@ export default class NextNodeServer extends BaseServer {
   public getRequestHandler(): NodeRequestHandler {
     const handler = super.getRequestHandler()
     return async (req, res, parsedUrl) => {
+      const normalizedReq = this.normalizeReq(req)
+      const normalizedRes = this.normalizeRes(res)
+      await maybeStartProfilingRequest(
+        normalizedReq as NodeNextRequest,
+        normalizedRes as NodeNextResponse
+      )
       return handler(this.normalizeReq(req), this.normalizeRes(res), parsedUrl)
```

Comment:
```suggestion
      return handler(normalizedReq, normalizedRes, parsedUrl)
```

Is it possible to avoid `this.normalize{Req,Res}` twice?